languages/julia/exercises/concept/methods.basic/.meta/design.md

```diff
@@ -0,0 +1,24 @@
+# Design
+
+## Goal
+
+The goal of this exercise is to teach the student the fundamentals of the Concept of methods in Julia, so that they are able to understand the basic structure of Exercism exercises.
+
+## Learning objectives
+
+- Know the two common ways to define a named function.
+  - `f(x) = ...`, `function f(x) ... end`
+- Know how to call a function.
+- Know about implicit `return`s.
+- Know how to define an integer.
+- Know how to use mathematical operators on integers.
+
+## Out of scope
+
+## Concepts
+
+The Concepts this exercise unlocks are:
```

Comment:
The concept this exercise unlocks should be added here.

Reply:
I don't know yet. It'll have to be added later.

Reply:
But you have written the exercise, right? I saw an example file and such. You would want to wait with specifying what Concepts this exercise unlocks to see what Concepts other exercises need?

Reply:
Yes, I've written it. But because it introduces nothing more than the bare minimum necessary to understand the structure of an exercise, I haven't thought about what the next exercise will be yet. It could unlock loops, arrays, variables, types, printing, or any other concept, really.

Reply:
Yep, that is precisely what it should be :) Example: [design.md](https://github.com/exercism/v3/blob/master/languages/csharp/exercises/concept/flag-enums/.meta/design.md#concepts) -> [config.json](https://github.com/exercism/v3/blob/master/languages/csharp/config.json#L33).

Reply:
Ah alright, I misunderstood that. I thought it was about the concepts that this concept opens up because it's a prerequisite for them. I'll update it to match the `config.json` once the other discussion about the concept slug itself has been resolved.

Reply:
Should be resolved by 3f7232a